src/lib/response.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const response = module.exports = {};
+
+response.sendJSON = (res, status, data) => {
+  res.writeHead(status, { 'Content-Type': 'application/json' });
+  // res.write requires a buffer or a string.
```

Comment:
I encourage comments, but try to get in the habit of taking them out before submissions.